model/riscv_insts_vext_utils.sail

```diff
@@ -179,8 +179,11 @@ function get_velem_quad(v, i) = v[4 * i + 3] @ v[4 * i + 2] @ v[4 * i + 1] @ v[4
 /* Divide the input bitvector into 4 equal slices and store them in vd starting at position 4*i */
 val write_velem_quad : forall 'n, 'n > 0 . (vregidx, sew_bitsize, bits('n), nat) -> unit
 function write_velem_quad(vd, SEW, input, i) = {
-  foreach(j from 0 to 3)
-    write_single_element(SEW, 4 * i + j, vd, slice(input, j * SEW, SEW));
+  foreach(j from 0 to 3) {
+    // write_single_element(SEW, 4 * i + j, vd, slice(input, j * SEW, SEW));
```

Comment:
Probably remove this comment, and maybe add a `// TODO: Change the type signature to enforce this assertion.`